docker/arm/Dockerfile.arm32v7

```diff
@@ -0,0 +1,31 @@
+# Dockerfile to be used in an ARMv7 environment like Raspverry PI 4
+FROM arm32v7/node:slim
+LABEL maintainer="Coderaiser, tea-mo903"
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app/
+
+RUN apt-get update && apt-get -y install python3 make g++
+
+ENV PYTHON python3
```

Comment:
Could you please remind me why do we need to set PYTHON env variable :)?

Reply:
It is need for `gritty`, which again depends on `node-pty` and that needs `python`. [Read Here](https://github.com/coderaiser/cloudcmd/pull/268#issuecomment-571721247)

But as you see in [Line 19](https://github.com/tea-mo903/cloudcmd/blob/4ee695c1fde9d99b54e3e81d82198571eb620e88/docker/arm/Dockerfile.arm32v7#L19), python gets removed after the install again.